cli/cli.go

```diff
@@ -284,6 +290,31 @@ func fetchIndex() {
 	}
 }
 
+func installFromVersionFile() {
+	blob, err := ioutil.ReadFile(VERSION_FILE)
+	if err != nil {
+		fmtc.Println("Cannot read %s", VERSION_FILE)
+		exit(1)
+	}
+
+	version, err := readFirstWord(string(blob))
+	if err != nil {
+		fmtc.Println("Cannot find version in %s", VERSION_FILE)
+		exit(1)
+	}
+
+	fmtc.Println("Installing version %s from %s", version, VERSION_FILE)
+	installCommand(version)
+}
+
+func readFirstWord(body string) (string, error) {
+	matches := regexp.MustCompile(`^\s*(\S+)\s*`).FindStringSubmatch(body)
```

Comment:
Please avoid using `regexp` package for removing garbage from version file. All what you need is present in `strings` package,